CHANGELOG.md

```diff
@@ -31,6 +33,16 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 ### Technical Changes
 
 - Scanner V4 now uses [Red Hat's VEX files](https://security.access.redhat.com/data/csaf/v2/vex/) instead of the [CVE map](https://security.access.redhat.com/data/metrics/cvemap.xml) for vulnerability data related to non-RPM content inside of official Red Hat images.
+- ROX-27622: Move `SecurityPolicy` CRD to template directory in Helm chart. **All Helm users will need to take action!**
+  No action is needed for users that use the operator or `roxctl` to install StackRox.
+  This change makes the CRD simpler to maintain for users because it will now be automatically upgraded.
+  To avoid installation failure, Helm users need to apply the following changes to the CRD prior to upgrade:
+
+      kubectl annotate crd/securitypolicies.config.stackrox.io meta.helm.sh/release-name=stackrox-central-services
+      kubectl annotate crd/securitypolicies.config.stackrox.io meta.helm.sh/release-namespace=stackrox
```

Comment:
We do allow configuring the namespace to other than stackrox right? So this line will have to accommodate that.